observability-tools/src/datatypes.ts

```diff
@@ -12,7 +12,8 @@ export const schemaTestCaseLine = z.object({
   how_generated: z.string().optional(),
   features: z.record(z.any()),
   coverage: z.union([z.record(z.array(z.number())), z.literal("no_coverage_info"), z.null()]),
-  metadata: z.any()
+  metadata: z.any(),
+  timing: z.record(z.number()),
```

Comment:
Wasn't sure if this should be optional or not

Reply:
Yeah, probably should be. As it is I think our schema is probably not permissive enough.